playground/README.md

```diff
@@ -62,19 +62,47 @@ cd beam
 
 ## Run local environment using docker compose
 
-```
+### Router only
+
+Start:
+
+```bash
 cd beam
 ./gradlew playground:backend:containers:router:dockerComposeLocalUp
```

Comment:
Now that we are migrating to kubernetes, may we consider moving away from docker compose?  An alternative is minikube.  If you'd like a pair programming session to set this up, I'm more than willing.

Reply:
I have never been using Kubernetes in any way so it would take days to prepare for that session. Although I am too more than willing.

There may be benefits of minikube over local Docker, but this branch is more of a daily tool to us to develop against an undeployed backend. And the frontend does not care how the backend is started as long as it has a host and a port. We currently copy this configuration from one local branch to another. So it would just save time today if we had it in master.

One other reason to keep docker compose for awhile is that we already have it elsewhere, for instance in https://github.com/apache/beam/blob/master/playground/backend/containers/router/docker-compose.local.yml